flaskphiid/annotation.py

```diff
@@ -47,7 +54,20 @@ def from_annotations(anns):
         merged = MergedAnnotation()
         for ann in anns:
             merged.add_annotation(ann)
-        return merged
+        if merged.has_compatible_family_typelist:
```

Comment:
can we name this something like "is_not_unknown" since that's all it's checking?